Update backup.md (#718)
percona/pmm-doc: docs/using/backup.md

- [PMM Client](../setting-up/client/index.md) is installed and running on the node.

> - For MySQL 8.0+ the user that pmm-agent is using to connect to MySQL needs to have the BACKUP_ADMIN privilege for Xtrabackup to work.

Reviewer: ```suggestion
- For MySQL 8.0+, the user that pmm-agent uses to connect to MySQL must have the BACKUP_ADMIN privilege for Xtrabackup to work.
```
